jenkins_pipelines/manager_prs/Jenkinsfile_prs_changelog

```diff
@@ -41,7 +46,11 @@ pipeline {
         stage('Run changelo test') {
             steps {
                 echo 'Run changelog tests'
-                sh "${gitarro_changelog_test}"
+                sh "${gitarro_changelog_test};" +
+                   "CLRET=\$?;" +
+                   "PR_NUMBER=\$(grep 'GITARRO_PR_NUMBER:' ${env.WORKSPACE}/.gitarro_vars | cut -d ':' -f2);" +
+                   "if [ \$CLRET -eq 0 ]; then ${gitarro_enable_merging} -t /usr/bin/true -P \$PR_NUMBER; else ${gitarro_enable_merging} -t /usr/bin/false -P \$PR_NUMBER; fi;" +
```

Comment:
Just a not for the future: cloning the repo again just to run /usr/bin/true is a waste of time and resources.

It would be good if we could tell gitarro to avoid cloning, or if we could switch two statuses with one single test :-)